router/comments.go

```diff
@@ -43,7 +43,7 @@ func PostComments(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, err)
 	}
 	itemInfo := fmt.Sprintf("[%v](https://%v/items/%v)", item.Name, os.Getenv("HOST"), item.ID)
-	message := fmt.Sprintf("### コメントを投稿しました\n%v\n%v", itemInfo, comment.Text)
+	message := fmt.Sprintf("### @%s がコメントを投稿しました\n%s\n%s", comment.User.DisplayName, itemInfo, comment.Text)
```

Comment:
```suggestion
	message := fmt.Sprintf("### @%s がコメントを投稿しました\n%s\n%s", comment.User.Name, itemInfo, comment.Text)
```
`User`のモデルはtraQがもとになってます
一つ目の画像がtraQのもの、二つ目の画像がbooQのものです
![image](https://user-images.githubusercontent.com/48485650/124127392-a01b9f80-dab6-11eb-8218-6557f6342e80.png)
![image](https://user-images.githubusercontent.com/48485650/124127303-87ab8500-dab6-11eb-874c-ecd8b64eb690.png)

投稿するメッセージの`@hoge`は`displayName`じゃなくて`name`じゃないといけないのでそう修正してほしいということです

booQの`User`モデルはここにあります
https://github.com/traPtitech/booQ/blob/e3073b38aa7953cc11ec579552f11e694b92dea0/model/users.go#L8~L13